app/templates/services/_base_service_page.html

```diff
@@ -78,30 +78,40 @@
           ],
           field_headings_visible=False
         ) %}
-          {% call summary.row(complete=not question.answer_required) %}
+          {% call summary.row() %}
             {{ summary.field_name(question.label) }}
-            {% if question.answer_required %}
-              {% call summary.field() %}
-                {% if framework and framework.status == 'open' %}
-                    <a href="{{ url_for(".edit_service_submission", framework_slug=framework.slug, lot_slug=service_data.lot, service_id=service_id, section_id=section.id, _anchor=question.id) }}">Answer required</a>
+            {% if question.is_empty %}
+              {% if framework and framework.status == 'open' and section.edit_questions %}
+                {% if lot['oneServiceLimit'] %}
```

Comment:
It doesn't matter either way, but we tend to use dot notation in templates (ie `lot.oneServiceLimit`), since it's equivalent in Jinja.